README.md

```diff
@@ -270,6 +272,14 @@ A complete API definition is in the works. Meanwhile, you can learn how to you u
   - [`ipfs.object.patch.setData(multihash, data, [options, callback])`](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/OBJECT.md#objectpatchsetdata)
 - [pin (not implemented, yet!)](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/)
 
+#### `Security`
+
+- [key](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/KEY.md)
+  - `ipfs.key.gen(name, options, [callback])`
+  - `ipfs.key.list([callback])`
+  - `ipfs.key.rename(oldName, newName, [callback])`
+  - `ipfs.key.rm(name, [callback])`
+
```

Comment:
Add also the `crypto` API while you are at it

Reply:
crypto is NYI, there's not even a spec yet.

Reply:
There is the same spec that there is for key https://github.com/ipfs/specs/tree/master/keystore#interface

Reply:
The `crypt` command is not implemented in `go`. Also, I have issues with the proposed spec; which I would like to raise in another place.

Reply:
See https://github.com/libp2p/js-libp2p-keychain/pull/19